projects/control-service/projects/helm_charts/pipelines-control-service/templates/secret_pull_vdk_sdk_img.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if (include "shouldCreateVdkSdkDockerRepoSecret" .) }}
```

Comment:
Would be nice to keep the two secrets consistent - i.e. introduce a template helper similar to `shouldCreateVdkSdkDockerRepoSecret`, etc.

Reply:
Agreed, but I prefer not to refactor the old secret.